src/game/core/explore.js

```diff
@@ -16,16 +16,12 @@ import { Observable } from 'rx'
 
 import models from '../models'
 import { run } from './combat'
+import { weightedPool, pickRandom } from './utils'
```

Comment:
I don't like a module named `utils` because it's a place where people will throw stuff that they don't had patience to properly name. In this case, I'd prefer creating two modules called `pickRandom` and `weightedPool`.